tests/python/unittest/test_auto_scheduler_sketch_generation.py

```diff
@@ -0,0 +1,107 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+""" Test sketch generation. """
+
+import tvm
+from tvm import te, auto_scheduler
+
+from test_auto_scheduler_common import (matmul_auto_scheduler_test, conv2d_nchw_bn_relu_auto_scheduler_test,
+                                        max_pool2d_auto_scheduler_test, min_nm_auto_scheduler_test,
+                                        softmax_nm_auto_scheduler_test, softmax_abcd_auto_scheduler_test,
+                                        conv2d_winograd_nhwc_auto_scheduler_test)
+
+def print_sketches(sketches):
```

Comment:
I guess this is for debugging purpose? Should we move it to sketch since it might be useful for developers as well?

Reply:
Merged this with the `SketchPolicy::generate_sketches()`.